src/material-experimental/mdc-progress-spinner/progress-spinner.scss

```diff
@@ -0,0 +1,20 @@
+@import '@material/circular-progress/mixins.import';
+@import '../mdc-helpers/mdc-helpers';
+
+
+@include mdc-circular-progress-core-styles($query: $mat-base-styles-without-animation-query);
+
+.mat-mdc-progress-spinner {
+  // Explicitly set to `block` since the browser defaults custom elements to `inline`.
+  display: block;
+
+  .mdc-circular-progress {
+    // MDC circular spinner sets width and height to 48px by default.
```

Comment:
`// MDC circular spinner sets width and height to 48px by default, but we want it to expand to fill the host element`